docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -294,12 +328,16 @@ for your storage solution.
      - Description
      - Default value
    * - ``exchange.base-directories``
-     - The base directory URI location that the exchange manager uses to store
-       spooling data. Only supports S3 and local filesystems.
+     - Comma-separated list of URI locations that the exchange manager uses to
+       store spooling data. Only supports S3 and local filesystems.
      -
    * - ``exchange.encryption-enabled``
      - Enable encrypting of spooling data.
      - ``true``
+   * - ``exchange.compression-enabled``
```

Comment:
It is a configuration property (an old one, pre fault tolerant execution). It should be set in `config.properties`

Reply:
Moved this up to the higher-level FTE config properties, and added a recommendation down here in the exchange manager section to set this to `true`.